pgm-2.txt

```diff
@@ -0,0 +1,7 @@
+
+#!/bin/bash
+echo "hellooooooo"
+find *.log > z.txt
+find *.txt >> z.txt
+sudo mkdir /home/sudhir/zout
+tar -zczf matches.tar.gz /home/main
```

Comment:
the comment tar -zczf matches.tar.gz /home/main directly creates the tar.gz file so I used it here.